.github/workflows/storybook-publish.yml

```
@@ -36,6 +40,14 @@ jobs:
         working-directory: 'packages/react'
         run: yarn storybook:build
 
```

```suggestion
      - name: Create CNAME
        run: |
          touch storybook-static/.nojekyll
          touch storybook-static/CNAME
          echo "labs.carbondesignsystem.com" > storybook-static/CNAME
```

I think we might need this step if the root is gonna be cleared out and replaced with the root build files